dev-libs/aws-checksums/aws-checksums-0.1.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="Cross-Platform HW accelerated CRC32c and CRC32 with software fallbacks"
+HOMEPAGE="https://github.com/awslabs/aws-checksums"
+SRC_URI="https://github.com/awslabs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+DEPEND="dev-util/ninja"
```

Comment:
inheriting cmake-utils.eclass will grant you the ninja dependency here. So you can just remove DEPEND and RDEPEND from this ebuild, and from the other similar ebuilds in this PR.

Reply:
Got it!
And while on it, I also found an error with DEPEND/RDEPEND in the aws-sdk-cpp ebuild.